generator/views.py

```diff
@@ -56,3 +58,9 @@ def release_checklist(request, version):
 def securityrelease_checklist(request, pk):
     instance = get_object_or_404(SecurityRelease, pk=pk)
     return render_checklist(request, instance)
+
+
+@permission_required("generator.view_securityissue")
```

Comment:
Noticed we have two perms here, `view_securityrelease` and `view_securityissue`. Should the `securityrelease_checklist` view require both, or are we just going to depend on these always being assigned in tandem?